src/API/IEX.h

```diff
@@ -13,12 +13,12 @@ namespace API {
 class IEX {
 public:
   json getChart(const std::string symbol);
-  json getParsedJson(const std::string endpoint, const bool verifySSL=false);
+  virtual json getParsedJson(const std::string endpoint, const bool verifySSL = false);
```

Comment:
You can use `const std::string &` to avoid copies of the strings for all methods in this class.